README.md

```diff
@@ -8,6 +8,7 @@ This repository also contains the Zowe Node Client SDK. The SDK lets you leverag
 
 ## Contents  <!-- omit in toc -->
 
+ - [Early Access Features](#early-access-features)
```

Comment:
This link goes to the right document, but appears to go to the  "Build from source" link.

Reply:
I think a couple of the following links also go to the wrong topic.

Reply:
The link works fine for me. Maybe it behaves differently than expected when clicked in the PR diff view?
![github-readme-toc](https://user-images.githubusercontent.com/22344007/106316794-509d2300-623b-11eb-86be-2356411b2eb9.gif)

Reply:
Then I am not worried about it.